api_tests/registrations/views/test_registration_files_list.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from tests.json_api_test_app import JSONAPITestApp
+from api.base.settings.defaults import API_BASE
+from api_tests import utils as api_utils
+from tests.base import ApiTestCase
+from osf_tests.factories import (
+    AuthUserFactory,
+    ProjectFactory,
+    RegistrationFactory
+)
+
+@pytest.mark.django_db
+class TestRegistrationFilesList(object):
+
+    @pytest.fixture(autouse=True)
+    def setUp(self):
```

Comment:
Ah, looks like it's a known issue: https://github.com/pytest-dev/pytest-django/issues/297 , and that your workaround is correct: https://stackoverflow.com/questions/34089425/django-pytest-setup-method-database-issue